spec/features/editing_the_cma_case_finder_spec.rb

```diff
@@ -11,6 +14,7 @@
     log_in_as_editor(:cma_editor)
     stub_publishing_api_has_content([], hash_including(document_type: CmaCase.document_type))
     stub_publishing_api_has_content(organisations, hash_including(document_type: Organisation.document_type))
+    stub_any_support_api_call
```

Comment:
I think it's important to add a test here, that verifies what we're sending to Zendesk. I'd at least want to cover the requester email, the description, and the tag.

Reply:
I've added a scenario controller to cover this since it was easier to verify that the api had actually be called.